src/index.ts

```diff
@@ -4,20 +4,32 @@ import { exec } from "child_process";
 import { join } from "path";
 
 export default function run(commandArgs: string[]) {
-  const strykerConfPath = join(process.cwd(), "stryker.conf.js");
+
+  const args = commandArgs.slice(2);
+  let branch;
+  let path;
+
+  mapAllArgumentsToAnArray(args).forEach((element) => {
```

Comment:
Maybe we could avoid looping over the entire array and use `.filter` or `.find` for both seeking arguments ?